Source/Windows/Amg8833Camera/Amg8833Camera.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0-windows</TargetFramework>
```

Comment:
can we use net8.0-windows? is what most users will have installed at this point.